### PR TITLE
Make one .env in a  root and update Readme.md

### PR DESCRIPTION
**Pull request by OlehX**

### CODE REVIEW

**Comment by AoDev** [comment]
@OlehX The env.sample should be in the root folder as well and delete the one that is in fulcrum.

File: README.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+
+Run from the **contract repo folder**.
+
+**Start the fork**
+
+```
+brownie console --network mainnet-fork
+```
+
+**Setup the fork + useful commands**
```
Comment (AoDev) [inline README.md:67]: Better add the commands that setup the fork here as well.

**Comment by OlehX** [comment]
> @OlehX The env.sample should be in the root folder as well and delete the one that is in fulcrum.

No file has been created

**Comment by AoDev** [comment]
I see, I missed it because Git consider that the fulcrum one has been moved to the root folder

**Pull request merged**